revdep.R

```diff
@@ -1,6 +1,6 @@
 # Run by package maintainer via these entries in ~/.bash_aliases :
 #   alias revdepr='cd ~/build/revdeplib/ && R_LIBS_SITE=none R_LIBS=~/build/revdeplib/ _R_CHECK_FORCE_SUGGESTS_=false R_PROFILE_USER=~/GitHub/data.table/revdep.R R'
-#   alias revdepsh='cd ~/build/revdeplib/ && export R_LIBS_SITE=none && export R_LIBS=~/build/revdeplib/ && export _R_CHECK_FORCE_SUGGESTS_=false'
+#   alias revdepsh='cd ~/build/revdeplib/ && export TZ=UTC && export R_LIBS_SITE=none && export R_LIBS=~/build/revdeplib/ && export _R_CHECK_FORCE_SUGGESTS_=false'
```

Comment:
i feel like this is setting us up for trouble as it's common not to have `TZ` set and we may want this to help us see related errors, no?

Reply:
(I don't have `TZ` set btw)

Reply:
CRAN sets `TZ=UTC`.  The trouble in testing under local timezone is that there's no cross-platform way to do it.

Reply:
Although you don't have TZ set, your timezone is different to mine. I guess yours overlaps with the fixed timezone used by `CoSMoS` in a different way than mine, and that causes it to pass.